packages/idx/src/property/admin/search.filter.component.html

```diff
@@ -253,7 +255,8 @@ <h3>Display Options</h3>
                             <md-option value="2">SFARMLS</md-option>
```

Comment:
Don't we also need to the `0 => Exclusive` properties in this list?

Reply:
@chadwickmeyer 
they are there on lines 253 and 254: https://github.com/Sitetheory/stratus/pull/380/files/de474038f8fead08f915f74c8133916ee1da30e3#diff-11ef942603e7978abce4ece56d3a8d42R253-R254

this section will also be replaced with the services that the user has access to as well shortly